Src/stm32f4xx_it.c

```diff
@@ -252,6 +279,14 @@ void EXTI1_IRQHandler(void)
 		memcpy(P2_GPIOC_current, P2_GPIOC_next, 64);
 		memcpy(P1_GPIOC_current, P1_GPIOC_next, 64);
 
+		// copy the multitap bits too

```

Comment:
Let's talk about this. I think we can make it more efficient. I don't think we need memcpy calls and calculations for both P1_GPIOC_next_multitap and P1_GPIOC_next. The only difference would be a single bit if the D2 line is being used.